test/membrane/rtp/twcc_sender/congestion_control_test.exs

```diff
@@ -245,8 +245,7 @@ defmodule Membrane.RTP.TWCCSender.CongestionControlTest do
         )
 
       assert cc.state == :decrease
-      assert cc.a_hat >= 0.80 * rate
-      assert cc.a_hat <= 0.95 * rate
+      assert_in_delta cc.a_hat / rate, 0.85, 0.95
```

Comment:
Consider such an assertion

```elixir
   assert_in_delta 0.75, 0.8, 0.5
 ```
 
 It will pass, so I would have to write `assert_in_delta, x, 0.875, 0.75` which I think is less redable as we operate on strange `0.875`. The requirement to be between 0.8 and 0.95 is from the RFC draft.
 
 ```
    When an over-use is detected the system transitions to the decrease
   state, where the delay-based available bandwidth estimate is
   decreased to a factor times the currently incoming bitrate.

     A_hat(i) = beta * R_hat(i)

   beta is typically chosen to be in the interval [0.8, 0.95], 0.85 is
   the RECOMMENDED value.
   ```

Reply:
You're right, the explicit comparison is more readable since the values come from RFC. I guess you can add a comment referencing the part you quoted to make these values less magical